testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@rhlike_minion
+Feature: Add the Rocky 8 distribution custom repositories
+  In order to use Rocky 8 channels with Red Hat "modules"
+  As a SUSE Manager administrator
+  I want to filter them out to remove the modules information
+
+  Scenario: Download the iso of Rocky 8 DVD and mount it on the server
+    When I mount as "rocky-8-iso" the ISO from "http://minima-mirror.mgr.suse.de/pub/rocky/8/isos/x86_64/Rocky-x86_64-dvd.iso" in the server
```

Comment:
That's probably out of the scope.
But it will be really good to work in a utils method that in a smart way select a minima mirror from Nuremberg or from Provo. Or even choose a BV or a CI mirror.

Reply:
Yes, we can improve that over time. I just copied that scenario over from the CentOS 7 on in the BV.